### ISSUE TITLE
SameDiff TF import: error importing .pb graph generated with darkflow that was trained by darknet

### ISSUE THREAD

**Issue by androuino** [eclipse/deeplearning4j#6227]
Hi, I was trying to load a .pb file that was converted using **darkflow** `flow --model cfg/my-tiny-yolo-obj.cfg --load bin/my-tiny-yolo-obj.weights --savepb` that was trained using **darknet**. When I tried to use the `LoadTensorFlowMNISTMLP` class I got this error:

`org.nd4j.linalg.exception.ND4JIllegalStateException: No tensorflow op found for Pad possibly missing operation class?
    at org.nd4j.imports.graphmapper.tf.TFGraphMapper.mapNodeType(TFGraphMapper.java:495)
    at org.nd4j.imports.graphmapper.tf.TFGraphMapper.mapNodeType(TFGraphMapper.java:36)
    at org.nd4j.imports.graphmapper.BaseGraphMapper.importGraph(BaseGraphMapper.java:219)
    at org.nd4j.imports.graphmapper.BaseGraphMapper.importGraph(BaseGraphMapper.java:69)
    at org.nd4j.imports.graphmapper.BaseGraphMapper.importGraph(BaseGraphMapper.java:107)
    at org.deeplearning4j.examples.modelimport.tensorflow.LoadTensorFlowMNISTMLP.main(LoadTensorFlowMNISTMLP.java:50)
Exception in thread "main" org.nd4j.linalg.exception.ND4JIllegalStateException: Unknown format: /home/sem/IdeaProjects/dl4j-examples/dl4j-examples/target/classes/modelimport/tensorflow/my-tiny-yolo-obj.pb
    at org.nd4j.imports.graphmapper.BaseGraphMapper.importGraph(BaseGraphMapper.java:114)
    at org.deeplearning4j.examples.modelimport.tensorflow.LoadTensorFlowMNISTMLP.main(LoadTensorFlowMNISTMLP.java:50)`

**Comment by yangchengjian** [comment]
There's a same bug when I attempt integrate dl4j into my project.
env: spring-boot
dl4j.version: 1.0.0-beta
```txt
org.nd4j.linalg.exception.ND4JIllegalStateException: Unknown format: /var/opt/smartmes/frozen_model.pb
org.nd4j.imports.graphmapper.BaseGraphMapper.importGraph(BaseGraphMapper.java:114)
org.thingsboard.rule.engine.ml.TbTFNode.createGraph(TbTFNode.java:137)
org.thingsboard.rule.engine.ml.TbTFNode.init(TbTFNode.java:71)
org.thingsboard.server.actors.ruleChain.RuleNodeActorMessageProcessor.initComponent(RuleNodeActorMessageProcessor.java:117)
org.thingsboard.server.actors.ruleChain.RuleNodeActorMessageProcessor.start(RuleNodeActorMessageProcessor.java:58)
org.thingsboard.server.actors.service.ComponentActor.preStart(ComponentActor.java:57)
 akka.actor.Actor$class.aroundPreStart(Actor.scala:485)
akka.actor.UntypedActor.aroundPreStart(UntypedActor.scala:97)
akka.actor.ActorCell.create(ActorCell.scala:590)
akka.actor.ActorCell.invokeAll$1(ActorCell.scala:461) akka.actor.ActorCell.systemInvoke(ActorCell.scala:483)
akka.dispatch.Mailbox.processAllSystemMessages(Mailbox.scala:282)
akka.dispatch.Mailbox.run(Mailbox.scala:223)
akka.dispatch.Mailbox.exec(Mailbox.scala:234)
scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
```

**Comment by AlexDBlack** [comment]
@yangchengjian First, please upgrade to 1.0.0-beta3, there were lots of fixes in the 2 releases since 1.0.0-beta.
If the problem still occurs, please open a new issue and include your graph so we can reproduce this.
